dev-python/munkres/munkres-1.1.2.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{3_5,3_6} )
```

Comment:
Any chance you could test if python3.7 works too?

Reply:
Done. All of the tests pass for each version.